app/components/documents/document-preview/document-preview-sidebar.hbs

```diff
@@ -2,55 +2,83 @@
   @size="xlarge"
   @position="right"
   @skin="white"
-  class="auk-u-pb-6 auk-o-flex auk-o-flex--vertical auk-scroll-wrapper"
+  class="auk-u-pb-6 auk-o-flex auk-o-flex--vertical
+    {{if @isOpen "auk-scroll-wrapper"}}"
+  @isCollapsed={{not @isOpen}}
 >
   <Auk::Navbar>
     <Auk::Toolbar as |toolbar|>
-      <toolbar.Group @position="left" as |group|>
-        <group.Item>
-          <Auk::Tabs>
-            <Auk::Tab
-              {{on "click" (fn this.setActiveTab "details")}}
-              @active={{eq this.activeTab "details"}}
-            >
-              {{t "detail"}}
-            </Auk::Tab>
-            <Auk::Tab
-              {{on "click" (fn this.setActiveTab "signatures")}}
-              @active={{eq this.activeTab "signatures"}}
-            >
-              {{t "signatures"}}
-            </Auk::Tab>
-            <Auk::Tab
-              {{on "click" (fn this.setActiveTab "versions")}}
-              @active={{eq this.activeTab "versions"}}
-            >
-              {{t "versions"}}
-            </Auk::Tab>
-          </Auk::Tabs>
-        </group.Item>
-      </toolbar.Group>
+      {{#if @isOpen}}
+        <toolbar.Group @position="left" as |group|>
+          <group.Item>
+            <Auk::Tabs>
+              <Auk::Tab
+                {{on "click" (fn this.setActiveTab "details")}}
+                @active={{eq this.activeTab "details"}}
+              >
+                {{t "detail"}}
+              </Auk::Tab>
+              <Auk::Tab
+                {{on "click" (fn this.setActiveTab "signatures")}}
+                @active={{eq this.activeTab "signatures"}}
+              >
+                {{t "signatures"}}
+              </Auk::Tab>
+              <Auk::Tab
+                {{on "click" (fn this.setActiveTab "versions")}}
+                @active={{eq this.activeTab "versions"}}
+              >
+                {{t "versions"}}
+              </Auk::Tab>
+            </Auk::Tabs>
+          </group.Item>
+        </toolbar.Group>
+        <Auk::Toolbar::Group @position="right">
+          <Auk::Toolbar::Item>
+            <Auk::Button
+              @skin="borderless"
+              @icon="chevron-right"
+              @layout="icon-only"
+              {{on "click" @onCollapse}}
+            />
+          </Auk::Toolbar::Item>
+        </Auk::Toolbar::Group>
+      {{else}}
+        <Auk::Toolbar::Group @position="center">
+          <Auk::Toolbar::Item class="auk-o-flex auk-o-flex--horizontal-center">
+            <Auk::Button
+              @skin="borderless"
+              @icon="chevron-left"
+              @layout="icon-only"
+              {{on "click" @onOpen}}
+            />
+          </Auk::Toolbar::Item>
+        </Auk::Toolbar::Group>
```

Comment:
Dit zou anders kunnen.
De content (linkse toolbar) steken achter die isOpen flag.
En dan de hele blok met de logica voor die links/rechts verkleinen naar 1 button met ifs, zoals in
https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/d2f05cfb507a2e3d1cd97dfb1d29ccc4c004141b/app/components/publications/publication-sidebar.hbs#L14